<dec f='include/llvm-6.0/llvm/Support/Error.h' l='52' type='void llvm::ErrorInfoBase::log(llvm::raw_ostream &amp; OS) const'/>
<use f='include/llvm-6.0/llvm/Support/Error.h' l='58' u='c' c='_ZNK4llvm13ErrorInfoBase7messageB5cxx11Ev'/>
<ovr f='include/llvm-6.0/llvm/Support/Error.h' l='354' c='_ZNK4llvm9ErrorList3logERNS_11raw_ostreamE'/>
<use f='include/llvm-6.0/llvm/Support/Error.h' l='357' u='c' c='_ZNK4llvm9ErrorList3logERNS_11raw_ostreamE'/>
<ovr f='include/llvm-6.0/llvm/Support/Error.h' l='1040' c='_ZNK4llvm7ECError3logERNS_11raw_ostreamE'/>
<doc f='include/llvm-6.0/llvm/Support/Error.h' l='51'>/// Print an error message to an output stream.</doc>
